### ISSUE TITLE
Cannot make config changes or use seed-editor command

### ISSUE THREAD

**Issue by Skotavus** [jaredlll08/Fluxed-Crystals-Reborn#24]
Hi all,

I am currently running a Resonant Rise 3.2.7.1 modpack including FC2, and I am unable to use the /fluxedcrystals seed-editor command (the seed-from-current command works just fine).

Also, I have changed numerous configurations pertaining to number of shards needed per ingot in the refiners and saved the file correctly, but upon restarting the server there is no change to the number of shards being crafted.
There are two folders for Fluxed Crystals in the /config folder (one "fluxedcrystals" and one "fluxedCrystals"), I've edited the files in both and no difference was observed.

If I need to provide any info about my server instance, please let me know.

**Comment by jaredlll08** [comment]
Are you on server? if so, you need to make the change on both client and server, and restart the client and server
Also, the seed editor is only for the client side, doesn't work server side

**Comment by Skotavus** [comment]
Yes, I should have mentioned that it was a server, sorry.

Does that mean that changes to the config file need to be passed around between all the users to make them update?

**Comment by ghost** [comment]
Yes, because this is dealing with item creation we cannot sync the server
to client via configs, they will need to have the copy

**Comment by Skotavus** [comment]
If the reason that I am seeing no change in refiner requirements is that my client-side config file isn't changing, does that mean that individual users could simply set their configs to what they want/a mismatch and have a different set of values on their clients?

**Comment by ghost** [comment]
no there is a true up that happens when clients connect but it also
requires the same configs.  But if you want to test that I would love to
see what happens.

**Comment by Skotavus** [comment]
I've updated both the server and client-side configurations (setting nether quartz refiner amount to 8 as a test), and the requirements for the refiner have not changed (still set at 20).

**Comment by devryb** [comment]
I have the same issue, upon changing the tier for some of the seeds on the client and servers config no change is seen still.
